src/zzz_od/auto_battle/auto_battle_operator.py

```
@@ -154,34 +167,39 @@ def operate_periodically(self) -> None:
+                    any_done = True
+                if not self.is_running:
                     break
+                if self.auto_turn_interval > 0 and now - self.last_turn_time > self.auto_turn_interval:
+                    turn_op.execute()
+                    self.last_turn_time = now
+                    any_done = True
+
+                if not any_done:
+                    if self._stop_event.wait(timeout=0.2):
+                        break
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)